### ISSUE TITLE
Error: package or namespace load failed for ‘quanteda’

### ISSUE THREAD

**Issue by jjphorn** [kbenoit/quanteda#1071]
I upgraded quanteda to version 0.99.22 (along with other packages) and when I try to load quanteda I receive the following error message:

"Error: package or namespace load failed for ‘quanteda’ in namespaceExport(ns, exports):
 undefined exports: %&%, Cholesky, .SuiteSparse_version, Diagonal, .symDiagonal, .sparseDiagonal, .trDiagonal, Hilbert, KhatriRao, Matrix, MatrixClass, spMatrix, sparseMatrix, rsparsematrix, Schur, abIseq, abIseq1, rep2abI, band, bandSparse, bdiag, .bdiag, c.sparseVector, condest, onenormest, .asmatrix, .dsy2mat, .dsy2dsp, .dxC2mat, .T2Cmat, ..2dge, .dense2sy, .C2nC, .nC2d, .nC2l, .diag.dsC, .solve.dgC.chol, .solve.dgC.qr, .solve.dgC.lu, diagN2U, diagU2N, .diagU2N, .diag2tT, .diag2sT, .diag2mat, drop0, expand, expm, facmul, fac2sparse, fac2Sparse, forceSymmetric, T2graph, graph2T, anyDuplicatedT, uniqTsparse, isTriangular, isDiagonal, isLDL, is.null.DN, invPerm, lu, nearPD, nnzero, formatSpMatrix, formatSparseM, .formatSparseSimple, printSpMatrix, printSpMatrix2, qrR, rankMatrix, readHB, readMM, sparse.model.matrix, sparseVector, symmpart, skewpart, tril, triu, updown, pack, unpack, .updateCHMfacto
In addition: Warning message:
S3 methods ‘print.sparseSummary’, ‘print.diagSummary’, ‘c.abIndex’, ‘c.sparseVector’, ‘as.array.Matrix’, ‘as.array.sparseVector’, ‘as.matrix.Matrix’, ‘as.matrix.sparseVector’, ‘as.vector.Matrix’, ‘as.vector.sparseVector’ were declared in NAMESPACE but not found "

I removed quanteda 0.99.22 and tried to re-install the prior version 0.99.12 and received  this error message:

 ** preparing package for lazy loading
Warning: S3 methods 'print.sparseSummary', 'print.diagSummary', 'c.abIndex', 'c.sparseVector', 'as.array.Matrix', 'as.array.sparseVector', 'as.matrix.Matrix', 'as.matrix.sparseVector', 'as.vector.Matrix', 'as.vector.sparseVector' were declared in NAMESPACE but not found
Error in namespaceExport(ns, exports) : 
  undefined exports: %&%, Cholesky, .SuiteSparse_version, Diagonal, .symDiagonal, .sparseDiagonal, .trDiagonal, Hilbert, KhatriRao, Matrix, MatrixClass, spMatrix, sparseMatrix, rsparsematrix, Schur, abIseq, abIseq1, rep2abI, band, bandSparse, bdiag, .bdiag, c.sparseVector, condest, onenormest, .asmatrix, .dsy2mat, .dsy2dsp, .dxC2mat, .T2Cmat, ..2dge, .dense2sy, .C2nC, .nC2d, .nC2l, .diag.dsC, .solve.dgC.chol, .solve.dgC.qr, .solve.dgC.lu, diagN2U, diagU2N, .diagU2N, .diag2tT, .diag2sT, .diag2mat, drop0, expand, expm, facmul, fac2sparse, fac2Sparse, forceSymmetric, T2graph, graph2T, anyDuplicatedT, uniqTsparse, isTriangular, isDiagonal, isLDL, is.null.DN, invPerm, lu, nearPD, nnzero, formatSpMatrix, formatSparseM, .formatSparseSimple, printSpMatrix, printSpMatrix2, qrR, rankMatrix, readHB, readMM, sparse.model.matrix, sparseVector, symmpart, skewpart, tril, triu, updown, pack, unpack, .updateCHMfactor, .validateCsparse, writeMM, cBind, rBind
ERROR: lazy loading failed for package 'quanteda'
* removing 'C:/Users/User/Desktop/Research/r/quanteda'
Warning in install.packages :
  running command '"C:/PROGRA~1/R/R-34~1.2/bin/x64/R" CMD INSTALL -l "C:\Users\User\Desktop\Research\r" "C:/Users/User/Downloads/quanteda_0.99.12.tar.gz"' had status 1
Warning in install.packages :
  installation of package ‘C:/Users/User/Downloads/quanteda_0.99.12.tar.gz’ had non-zero exit status

Please let me know if you need me to provide anymore diagnostic information.

**Comment by kbenoit** [comment]
Try 
```r
update.packages(ask = FALSE)
```
because it looks like you need to update the **Matrix** package.

But we cannot tell you more without the results from `sessionInfo()`.

**Comment by jjphorn** [comment]
Your response time is impressive! Ultimately I had to re-install the Matrix package and that seemed to fix the problem. Thank you!!

**Comment by samanta-varela** [comment]
Hello, I installed quanteda on my recently updated version of R, and I´m trying to load quanteda, but I get the following error: Error: package or namespace load failed for ‘quanteda’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called ‘ggplot2’
In addition: Warning message:
package ‘quanteda’ was built under R version 3.5.3 

Can anybody help  please?

**Comment by kbenoit** [comment]
@samanta-varela Sounds like you need to install **ggplot2**.

`Sys.info()` output?

**Comment by Ericarrangement** [comment]
Hi, when I tried to load 'quanteda', I got the error:
Package version: 3.2.0
Unicode version: 13.0
ICU version: 69.1
Error: package or namespace load failed for ‘quanteda’:
 .onAttach failed in attachNamespace() for 'quanteda', details:
  call: qatd_cpp_tbb_enabled()
  error: function 'Rcpp_precious_remove' not provided by package 'Rcpp'
In addition: Warning messages:
1: package ‘quanteda’ was built under R version 4.0.5 
2: In stringi::stri_info() :
  Your native charset does not map to Unicode well. This may cause serious problems. Consider switching to UTF-8.
3: In stringi::stri_info() :
  Your current locale is not in the list of available locales. Some functions may not work properly. Refer to stri_locale_list() for more details on known locale specifiers.
4: In stringi::stri_info() :
  Your native charset does not map to Unicode well. This may cause serious problems. Consider switching to UTF-8.
5: In stringi::stri_info() :
  Your current locale is not in the list of available locales. Some functions may not work properly. Refer to stri_locale_list() for more details on known locale specifiers.




My sessionInfo() output is:

R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 16299)

Matrix products: default

locale:
[1] LC_COLLATE=Chinese (Simplified)_China.936 
[2] LC_CTYPE=Chinese (Simplified)_China.936   
[3] LC_MONETARY=Chinese (Simplified)_China.936
[4] LC_NUMERIC=C                              
[5] LC_TIME=Chinese (Simplified)_China.936    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] Rcpp_1.0.6

loaded via a namespace (and not attached):
 [1] compiler_4.0.3     magrittr_2.0.1     Matrix_1.2-18      tools_4.0.3       
 [5] fastmatch_1.1-3    tinytex_0.32       stringi_1.6.2      grid_4.0.3        
 [9] quanteda_3.2.0     xfun_0.24          stopwords_2.3      RcppParallel_5.1.5
[13] lattice_0.20-41   


Can anyone help?
